src/registries/functionsRoutersRegistry.ts

```diff
@@ -0,0 +1,68 @@
+export const functionsRoutersRegistry = {
+  ethereum: {
+    contractAddress: "0x65Dcc24F8ff9e51F10DCc7Ed1e4e2A61e6E14bd6",
```

Comment:
recommend a more descriptive name -- routerAddress?

Reply:
Since the registry is called the "functionsRoutersRegistry" then I think the name "contractAddress" to describe "address of a contract in the Functions Router registry" should be enough